_data/projects/meds.yml

```diff
@@ -0,0 +1,12 @@
+name: *Emeds*
```

Comment:
The `*` values in this template are all not required, and it's confusing the CI build.

Please edit the file to remove them